sdk/anomalydetector/ai-anomaly-detector-rest/README.md

```diff
@@ -0,0 +1,59 @@
+# Azure AnomalyDetectorRest client library for JavaScript
+
+AnomalyDetector Rest Client
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
```

Comment:
@joheredi do we not have links to product/service documentation page for rest-level-clients?